cgmanifest.json

```
@@ -22875,8 +22875,8 @@
         "type": "other",
         "other": {
           "name": "rsyslog",
-          "version": "8.2108.0",
-          "downloadUrl": "http://www.rsyslog.com/files/download/rsyslog/rsyslog-8.2108.0.tar.gz"
+          "version": "8.2204.1",
+          "downloadUrl": "http://www.rsyslog.com/files/download/rsyslog/rsyslog-8.2204.1.tar.gz"
```

Nit: can we flip this to `https`
```suggestion
          "downloadUrl": "https://www.rsyslog.com/files/download/rsyslog/rsyslog-8.2204.1.tar.gz"
```